src/ol/control/zoomslidercontrol.js

```diff
@@ -185,11 +184,21 @@ ol.control.ZoomSlider.prototype.setMap = function(map) {
  */
 ol.control.ZoomSlider.prototype.initSlider_ = function() {
   var container = this.element;
-  var containerSize = goog.style.getSize(container);
+  var containerSize = {
+    width: container.offsetWidth, height: container.offsetHeight
+  };
 
   var thumb = container.firstElementChild;
-  var thumbMargins = goog.style.getMarginBox(thumb);
-  var thumbBorderBoxSize = goog.style.getBorderBoxSize(thumb);
+  var thumbComputedStyles = window.getComputedStyle(thumb);
+  var thumbMargins = {
+    left: parseFloat(thumbComputedStyles['marginLeft']),
+    right: parseFloat(thumbComputedStyles['marginRight']),
+    top: parseFloat(thumbComputedStyles['marginTop']),
+    bottom: parseFloat(thumbComputedStyles['marginBottom'])
+  };
+  var thumbBorderBoxSize = {
+    width:thumb.offsetWidth, height: thumb.offsetHeight
+  };
```

Comment:
Like in my suggested commit for `src/ol/map.js`, the `thumbMargins` and `thumbBorderBoxSize` object literals could be avoided.